yamale/tests/fixtures/types_good_data.yaml

```diff
@@ -9,3 +9,7 @@ map:
     hello: 1
     another: "hi"
 null: null
+defult_day: 2010-01-01
```

Comment:
I like that you just switched the default and custom between the good and bad data files....nice.